<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TransformEdgesWidget</class>
 <widget class="QWidget" name="TransformEdgesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>307</width>
    <height>218</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Transform Edges</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="radioButtonMakeComplete">
       <property name="toolTip">
        <string>This option connects all nodes of the given graph. The new edges do not have any preset weights.</string>
       </property>
       <property name="text">
        <string>Complete Graph</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButtonEraseEdges">
       <property name="toolTip">
        <string>This option removes all edges from the given graph.</string>
       </property>
       <property name="text">
        <string>Remove all Edges</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButtonReverseEdges">
       <property name="toolTip">
        <string>This option reverses all edges of a directed graph. Note that this does not apply to undirected graphs.</string>
       </property>
       <property name="text">
        <string>Reverse all Edges</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButtonMakeSpanningtree">
       <property name="toolTip">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'DejaVu Sans'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;This option executes &lt;span style=&quot; font-style:italic;&quot;&gt;Prim's Minimal Spanning Tree Algorithm&lt;/span&gt; at the graph. All edges without given weights are assumed to have weights equals 1. Directed and undirected graphs are processed accordingly.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>Spanning Tree Transformation</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButtonEraseSelfEdges">
       <property name="text">
        <string>Remove all Self Edges</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttons">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
